elements/pfe-styles/src/pfe-base.scss

```diff
@@ -66,14 +75,10 @@ th {
   text-align: left;
 }
 
-// Patternfly base styles
-*,
-*::before,
-*::after {
-  box-sizing: border-box;
-}
+// PatternFly Styles
 
 html {
+  @include pfe-set-broadcast-theme(light);
```

Comment:
this is really the point of this whole MR, the rest is just proving that it works